backend/tarsy/controllers/system_controller.py

```
@@ -23,3 +27,177 @@ async def get_system_warnings() -> List[SystemWarning]:
     """
     warnings_service = get_warnings_service()
     return warnings_service.get_warnings()  # Pydantic handles serialization
+
+
+@router.get("/mcp-servers", response_model=MCPServersResponse)
+async def get_mcp_servers(request: Request) -> MCPServersResponse:
```

Ruff (0.14.3)

Unused function argument: `request`

(ARG001)